Automattic/wp-calypso
# Marketplace Thank You: Update plugin card styles

Resolves #74740

**Depends on:**
- https://github.com/Automattic/wp-calypso/issues/74739

Once the following PR is merged. There should not be a border-top in the first plugin card:
- https://github.com/Automattic/wp-calypso/issues/74739

That will remove the extra section created when there are no themes

## Proposed Changes

* Update styles for Plugin cards in Marketplace Thank You page

## Testing Instructions



* Go to `/plugins:siteId` and purchase a couple of Marketplace plugins
* Make sure the design of the Thank You page plugin cards matches 5ddl1WFTEnaIatL2WlZp4k-fi-2596%3A11300

|Before | After|
|-------|------|
| ![CleanShot 2023-03-22 at 18 02 35@2x](https://user-images.githubusercontent.com/3519124/226982101-739e1213-9267-4fbe-b199-8c9f5cb35fa2.png) | ![CleanShot 2023-03-22 at 18 07 28@2x](https://user-images.githubusercontent.com/3519124/226983343-22474f4b-d6c6-45ec-9c90-dcf0ee9aa955.png)    |

## Pre-merge Checklist
